app/models/api/v3/contextual_layer.rb

```diff
@@ -3,6 +3,8 @@ module V3
     class ContextualLayer < BaseModel
       belongs_to :context
 
+      has_many :carto_layer
```

Comment:
Let's make the relation name plural please, this is strange :) We also need to consider how the front-end will handle the fact that different years could have different layers. I guess we could either parametrise the call to `map_layers` with year, and then we could filter out only relevant layers, or if we're serving all layers we need to let front-end know which year range each is for - I believe the `years` column is currently not serialised.

Reply:
relationship renamed.

As handling different layers for different years: that will require a bit more thought, but maybe that can be done on a separate PR